students/GKim/lesson01/maketags.py

```diff
@@ -0,0 +1,18 @@
+# The web is built with HTML strings like "<i>Yay</i>" which draws Yay as italic text. 
+# In this example, the "i" tag makes <i> and </i> which surround the word "Yay". 
+# Given tag and word strings, create the HTML string with tags around the word, e.g. "<i>Yay</i>".
+
+
+# make_tags('i', 'Yay') → '<i>Yay</i>'
+# make_tags('i', 'Hello') → '<i>Hello</i>'
+# make_tags('cite', 'Yay') → '<cite>Yay</cite>'
+
+def make_tags(tag, word):
```

Comment:
note for the future -- we'll learn about string formatting in a couple weeks -- you may want to come back to this then.

Reply:
is anything wrong with it?

Reply:
I guess I could have done this
def make_tags(tag, word):
   return "{}{}{}{}{}{}{}".format("<",tag,">",word,"</",tag,">")

Reply:
actually, what you want to do with string formatting, is put the text in teh string:

"<{}>{}</{}>".format(tag, word, tag)

(untested ...)

Reply:
doah!  Ah, ok.  It's all coming back.  Getting my mind back into python mode after about 3 weeks off.